sklearn/feature_selection/univariate_selection.py

```diff
@@ -211,16 +221,17 @@ def chi2(X, y):
 
 
 def f_regression(X, y, center=True):
-    """Univariate linear regression tests
+    """Univariate linear regression tests.
 
     Quick linear model for testing the effect of a single regressor,
     sequentially for many regressors.
 
     This is done in 3 steps:
-    1. the regressor of interest and the data are orthogonalized
-    wrt constant regressors
-    2. the cross correlation between data and regressors is computed
-    3. it is converted to an F score then to a p-value
+
+    1. The regressor of interest and the data are orthogonalized
+    wrt constant regressors.
```

Comment:
If you indent here, it becomes a valid restructured text list. Which would be a "'good" thing.